.gitlab-ci.yml

```diff
@@ -43,44 +75,24 @@ ci-image:
 ########################################################################################################################
 
 format:
-  stage: test
-  except:
-    refs:
-      - schedules
-  tags: ['runner:main', 'size:large']
-  image: $CI_IMAGE
+  extends: .test-configuration
```

Comment:
FMU this removes the `format` job from release branches. I think it's worth keeping it, because it applies on the CHANGELOG too.

Reply:
Ah right, the tags will trigger another pipeline. It looks fine then!